src/components/navigation/MetaMenu.tsx

```diff
@@ -15,35 +14,40 @@ interface MetaMenuLink {
 
 export default function MetaMenu(props: MetaMenuProps) {
   const { navigation, pagealias } = props
-  const links = navigation
+
+  const activeRef = useRef(null)
+  const containerRef = useRef(null)
```

Comment:
Should we remove these two or are you planning to use them later on? (because they seem unused here)

Reply:
thank you! Nice catch, actually forgot to move some code.